x-pack/examples/alerting_example/public/alert_types/astros.tsx

```diff
@@ -94,23 +94,23 @@ export function getAlertType(): AlertTypeModel {
 
 export const PeopleinSpaceExpression: React.FunctionComponent<PeopleinSpaceParamsProps> = ({
   alertParams,
-  setAlertParams,
+  setRuleParams,
   errors,
 }) => {
   const { outerSpaceCapacity = 0, craft = Craft.OuterSpace, op = Operator.AreAbove } = alertParams;
 
   // store defaults
   useEffect(() => {
     if (outerSpaceCapacity !== alertParams.outerSpaceCapacity) {
-      setAlertParams('outerSpaceCapacity', outerSpaceCapacity);
+      setRuleParams('outerSpaceCapacity', outerSpaceCapacity);
```

Comment:
Getting 

```
Cannot read properties of undefined (reading 'outerSpaceCapacity')
    at PeopleinSpaceExpression (astros.tsx:76)
```

when opening this rule param expressions. Think it should be solved by changing `alertParams` to `ruleParams`

Reply:
Yes you are right, thank you.